devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -16,8 +16,10 @@
 #include <sstream>
 #include <string>
 #include <tuple>
+#include <regex>
```

Comment:
Is regex needed?

Reply:
No, fixed in [d92f7f4](https://github.com/ami-iit/bipedal-locomotion-framework/pull/796/commits/d92f7f4392f6d2c5b42a7494948cb755fe16bf80)